troubleshooting.md

```diff
@@ -0,0 +1,570 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Bootstrap VM issues](#bootstrap-vm-related-issues)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Node Related Issues](#node-related-issues)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+**Pre-requisites:** 
+
+First and foremost the BM installations follows a general architecture:
+* provisioner node 
+* master node 
+* worker node
+
+Before running the installation, please make sure you have a cleaned the previous files/manifests.
+
+# Bootstrap VM related issues 
+
+* Cannot login to the Bootstrap VM
+```
+$ ssh core@172.22.0.2
+Permission denied (publickey,password,keyboard-interactive).
+```
+
+Generally if the SSH keys are not updated in `install-config.yaml` file you cannot login to the bootstrap VM, add/update the correct SSH keys. 
+
+Make sure this file has the SSH keys of the provisioner node, if not you can't login into any of the nodes after deployment is completed.
+
+After the installation is triggered, Check/verify bootstrap VM is up and running(may take few minutes) from Provisioner node: 
+
+```
+[kni@provisioner]$ sudo virsh list
+ Id    Name                           State
+----------------------------------------------------
+ 12    kni7-xf6fq-bootstrap           running
+```
+
+If there is no sign of Bootstrap VM running on the provioner node, check if the libvirtd is running(if not, start the service) on the provisioner node. 
+
+```
+systemctl status libvirtd
+● libvirtd.service - Virtualization daemon
+   Loaded: loaded (/usr/lib/systemd/system/libvirtd.service; enabled; vendor preset: enabled)
+   Active: active (running) since Mon 2020-02-03 13:08:51 EST; 2 weeks 0 days ago
+     Docs: man:libvirtd(8)
+```
+
+If re-running the deployment, please make sure to delete all the older VM including its volume as shown below, if not the bootstrap VM will not get a IP of `172.22.0.2` through provisioning dnsmasq and causes failure in further deployment process:
```

Comment:
For new troubleshooting pieces relating to the subject (in this case bootstrap VM), create a header for better separation.

Reply:
Created the sub-section as suggested.